### PR TITLE
Favouritecard mgmt

### PR DESCRIPTION
**Pull request by Aristotia**
legend finished, css done on slideforcards and link to game added

### CODE REVIEW

**Review by Pierre-Berger — APPROVED** [review]
ggggggggggggg

File: frontend/src/assets/css/DisplayProximityStation.css [hunk 1]
```diff
@@ -58,6 +60,7 @@
   height: 25%;
   margin: 0.5rem;
   gap: 5px;
+  padding-bottom: 00.5rem;
```
[the remark this answers is not in the document]
Comment (Aristotia) [reply frontend/src/assets/css/DisplayProximityStation.css]: fixed

[the remark this answers is not in the document]
Comment (Pierre-Berger) [reply frontend/src/assets/css/DisplayProximityStation.css]: Alt Text

**Pull request merged**